docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -199,6 +199,9 @@ The following table describes {ref}`catalog session properties
 * - `parquet_writer_page_size`
```

Comment:
what if the data is written by non-Trino writer? What is the root cause here (could you extend commit description)?